input/fsh/SD_Extensions.fsh

```diff
@@ -59,9 +63,38 @@ Description: "Dose parameters prescribed for one radiotherapy volume."
   * ^short = "Number of Prescribed Fractions"
   * ^definition = "The number of Fraction in which dose to this volume is prescribed. See also extension RadiotherapyFractionsPrescribed which is used instead if fractions are not per volume, e.g. in Phase Prescriptions or Plans."
 
+Extension: RadiotherapyDosePlannedToVolume
+Id: codexrt-radiotherapy-dose-planned-to-volume
+Title: "Dose Planned to Volume"
+Description: "Dose parameters planned for one radiotherapy volume."
+* . ^short = "Planned Dose to a Dose Reference"
+* extension contains
+    volume 1..1 MS and
+    totalDosePlanned 0..1 MS and
+    fractionsPlanned 0..1 MS and
+    fractionDosePlanned 0..1 MS
+* extension[volume]
+  * value[x] only Reference(RadiotherapyVolume)
+  * ^short = "Volume to which Dose is Planned"
+  * ^definition = "A BodyStructure resource representing the body structure treated, for example, Chest Wall Lymph Nodes."
+* extension[totalDosePlanned]
+  * value[x] only Quantity
+  * valueQuantity = UCUM#cGy
+  * ^short = "Total Planned Radiation Dose"
+  * ^definition = "The total dose planned to this volume within the scope of this ServiceRequest."
+* extension[fractionsPlanned]
+  * value[x] only positiveInt
+  * ^short = "Number of Planned Fractions"
+  * ^definition = "The number of Fraction in which dose to this volume is planned. See also extension RadiotherapyFractionsPlanned which is used instead if fractions are the same for all volumes, i.e. in Planned Phases or Plans."
+* extension[fractionDosePlanned]
+  * value[x] only Quantity
+  * valueQuantity = UCUM#cGy
+  * ^short = "Radiation Dose Planned per Fraction"
+  * ^definition = "The dose Planned per Fraction to this volume."
+
 Extension: RadiotherapyEnergy
 Id: codexrt-radiotherapy-energy
-Title: "(Future) Radiotherapy Energy"
+Title: "Radiotherapy Energy"
 Description: "The radiation energy, preferably specified as decimal in MV. Can alternatively be represented as string or CodeableConcept."
 * . ^short = "Radiotherapy Energy"
 * value[x] ^short = "The radiation energy as decimal in MV. Can alternatively be represented as string or CodeableConcept."
```

Comment:
Why do you want to leave the value type open?   how would it be represented as a codeable concept?
Shouldn't it be limited to a Quantity?   Like we do for dose?  With MV as the required units?

Reply:
That was biased by the fact that in our system the Energy is an enumeration of selectable energies. We have to discuss in the terminology call if we allow only for discrete coded values for a ValueSet, so one cannot send 9.999999MV or 10.004MV, or define a precision, or don't care too much. Another use for codes would be to have the same extension for energy or isotope, but I never changed the description and in the meantime I think it should be different extensions. Let's also discuss that in the call.

Reply:
Anyway, fixed to only Quantity for now to be inline with the description and scope that is only energy, not isotope so far.